### ISSUE TITLE
Fix a11y audit for progressbar

### ISSUE THREAD

**Issue by simonihmig** [kaliber5/ember-bootstrap#1520]
Missing label: https://dequeuniversity.com/rules/axe/4.2/aria-progressbar-name?application=axeAPI
